.drone.yml

```diff
@@ -131,7 +131,7 @@ pipeline:
         include: tag
       matrix:
         COMPOSER_BOUNDARY: highest
-        PHP_VERSION: 7.4
+        PHP_VERSION: "8.0"
```

Comment:
```suggestion
        PHP_VERSION: 8.0
```

Reply:
No, the artifact is not getting built because drone converts 8.0 to a regular number i.e. 8 and not a php version. Check previous tag builds where the steps for creating the artifact were not called.